examples/example3/README.md

```diff
@@ -0,0 +1,40 @@
+## Example 3 – Using virtual resources and collection in a master-agent set-up.
+
+
+This is an (almost) complete example for a master-agent set-up using virtual resources in Puppet.
```

Comment:
@xaque208 could you elaborate a little more on that? are you talking about `icinga2::object::service` in general, suggesting it should never be used as an exported resource, or are you talking about something more specific to this _example3_ ?